torch/optim/adamw.py

```diff
@@ -479,7 +479,7 @@ def _multi_tensor_adamw(
     if len(params) == 0:
         return
 
-    if capturable:
+    if not torch._utils.is_compiling() and capturable:
```

Comment:
Will inductor auto move params and state_steps to be CUDA tensors?

Reply:
no but it will partition the graph if they're not, so there shouldn't be any issue here